package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "replace-in-file",
-  "version": "3.2.0",
+  "version": "3.3.0",
```

Comment:
You don't need to bump the package version, I'll do that when I release a new version